Problem1.py

```diff
@@ -0,0 +1,37 @@
+"""
+740. Delete and Earn
+Time Complexity - O(n)
+Space Complexity - O(10**4) worst case
```

Comment:
You don't have to consider the maximum possible input. Specify your time and space complexity in term of input. if there are n inputs then space complexity would be n or n**2 depending on the algorithm. Here it is n.